man/systemd.network.xml

```diff
@@ -2450,6 +2450,14 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ControlledDelayPacketLimit=</varname></term>
+        <listitem>
+          <para>Specifies the hard limit on the real queue size. When this limit is reached, incoming packets are
+          dropped. Defaults to unset and kernel's default is used.</para>
```

Comment:
please say the the unit, i.e. "…hard lmit on the queue size in number of packets…" or so

Reply:
document valid range pls

Reply:
`qdisc codel 8002: dev dummy99 root refcnt 2 limit 4294967294p target 5.0ms interval 100.0ms`
Yes it supports full range